var/spack/repos/builtin/packages/zfp/package.py

```diff
@@ -162,3 +162,37 @@ def cmake_args(self):
                 args.append("-DCMAKE_CUDA_FLAGS=-arch sm_{0}".format(cuda_arch[0]))
 
         return args
+
+    def test_smoke_test(self):
+        """Build and run ctests"""
+        spec = self.spec
+        smoke_test_dir = self.test_suite.current_test_data_dir.tests
```

Comment:
Since you're saving files from the build, this should be `self.test_suite.current_test_cache_dir.tests`.

Reply:
We keep those separate from files that may be added in a `test` subdirectory of the package repository to reduce the risk of collisions.